akka-docs/src/main/paradox/cluster-sharding.md

```diff
@@ -130,7 +130,7 @@ Java
 @@@ div { .group-scala }
 
 A more comprehensive sample is available in the
-tutorial named [Akka Cluster Sharding with Scala!](https://github.com/typesafehub/activator-akka-cluster-sharding-scala).
+tutorial named [Akka Cluster Sharding with Scala](https://github.com/typesafehub/activator-akka-cluster-sharding-scala).
```

Comment:
That is a very! old sample. We should link to a newer sample. https://github.com/akka/akka-samples/tree/2.5/akka-sample-sharding-scala

Look at how we link to such samples in other places. I think the link is https://developer.lightbend.com/start/?group=akka&project=akka-samples-cluster-sharding-scala

There is also a corresponding Java sample

Reply:
Perfect, will update!